build_rpm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+git checkout -- setup.cfg
+rm -rf build dist *.egg *.egg-info
```

Comment:
As good practice in shell, we should not suppose the current running directory is just the directory where `build_rpm.sh` is.

We should use the following way to get the absolute path to build_rpm.sh.
```

cdir=`cd $(dirname $0); pwd`
(
cd $cdir
# you logic here. 
)
```

In that way, we can run build_rpm.sh anywhere, like ./build_rpm.sh works, /path/to/build_rpm.sh also can work..
I don't need to navigate to that folder to run the script..